src/main/java/org/openhab/binding/zwave/internal/protocol/ZWaveNode.java

```diff
@@ -1486,7 +1486,10 @@ public void run() {
             }
 
             // Tell the device to go back to sleep.
-            logger.debug("NODE {}: No more messages, go back to sleep", getNodeId());
+            if (isInitializationComplete() == true) {
+                logger.debug("NODE {}: No More Messages, Go back to sleep", getNodeId());
+                }
+                logger.warn("NODE {}: Initialization or Heal not complete, but sleep anyway, state {}", getNodeId(), getNodeInitStage());
```

Comment:
Aren't you missing an `else` statement, or should it always log a warning?

Reply:
So I'm not an expert on the subject, but I'm running this and the WARN only appears if "false".  It might be an allowed java shortcut with a boolean IF argument?  I could add an "else", if this is approved to merge.